### ISSUE TITLE
Exclude file from recursive search 

### ISSUE THREAD

**Issue by Sergei-Rudenkov** [aws-cloudformation/cfn-python-lint#531]
*cfn-lint version:  v0.10.2*

I have plenty of CF template files in `iac` folder. I would like them to be recursively linting:
```
templates:
- iac/CloudFormation/**/*.yaml
```

But I would like also to ignore some files from directory. Do you support something like:
```
ignore-templates:
- iac/CloudFormation/notCfTempl.yaml
```

**Comment by fatbasstard** [comment]
@Sergei-Rudenkov Quick question... The file you want to exclude, are that JSON/YAML file that are no CloudFormation files? Or also (valid) CloudFormation files that you'd like to exclude from the list?

**Comment by theslloyd** [comment]
A second vote for this. Particularly frustrating that I can't exclude my buildspec.yml (for AWS CodeBuild) so that I can run from the root directory of my repository. It can be avoided by specifying and retaining all CloudFormation files in a particular directory. Would be a nice enhancement to see for an otherwise great little tool!

**Comment by kddejong** [comment]
Released with v0.16.0
